src/pages/Chat/Chat.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import {
+  IChatUser,
+  useGetRoomsQuery,
+} from "services/notifications/setNotificationsAPI";
+import { IUser, useGetOwnUserQuery } from "services/user/setUserAPI";
+import { Block, Container, Notification } from "pages/Chat/Chat.styles";
+import { useTranslation } from "react-i18next";
+import ChatRoomsList from "./ChatRoomsList/ChatRoomsList";
+import ChatRoom from "./ChatRoom/ChatRoom";
+
+export interface IRoomUsers {
+  freelancer: IChatUser;
+  client: IChatUser;
+}
+
+function Chat() {
+  const { t } = useTranslation();
+
+  const { data: roomsList } = useGetRoomsQuery();
+
+  const { data: user } = useGetOwnUserQuery();
+
+  const [roomId, setRoomId] = useState<number | null>(null);
+  const [jobNameForHeader, setJobNameForHeader] = useState<string>("");
+  const [roomUsers, setRoomUsers] = useState<IRoomUsers>();
+
+  const onClickHandler = (id: number, jobName: string) => {
+    setRoomId(id);
+    setJobNameForHeader(jobName);
+  };
+
+  const defineOpponentsNameAndAvatar = (
+    freelancer: IChatUser,
+    client: IChatUser,
+    currentUser: IUser
+  ): { name: string; avatar: string } => {
+    if (+currentUser.id!! === freelancer.id) {
```

Comment:
one exclamation mark is not enough for ts?

Reply:
+